src/natcap/invest/validation.py

```diff
@@ -356,14 +319,15 @@ def check_vector(filepath, required_fields=None, projected=False,
     Args:
         filepath (string): The path to the vector on disk.  The file must exist
             and be readable.
-        required_fields=None (list): The string fieldnames (case-insensitive)
-            that must be present in the vector layer's table.  If None,
-            fieldnames will not be checked.
+        fields=None (list[str]): A list of regex-compilable patterns
```

Comment:
I think  we decided against regex for now? Does that apply here?

Reply:
good catch, updated these docstrings in dc72cfc1